frontend/src/components/common/FileUpload.js

```diff
@@ -28,13 +36,16 @@ class FileUpload extends React.Component {
   }
 
   render() {
+    const { maxFileSize, acceptedFileTypesMap } = this.props;
```

Comment:
We've decided to never use destructuring in props assignment. Not all existing files have been updated, but they're being refactored as we touch them. The goal is to avoid introducing new linting errors/warnings while gradually removing existing ones.

Reply:
Has been removed.